.eslintrc

```diff
@@ -1,75 +1,27 @@
 {
-    "env": {
-        "node": true,
-        "browser": true,
-        "mocha": true
-    },
-    "globals": {
-        "navigator",
-        "$serverSide",
-        "$isiOS",
-        "$isAndroid",
-        "$isBrowser"
-    },
-    "parser": "babel-eslint",
-    "parserOptions": {
-      "ecmaVersion": 7,
-      "ecmaFeatures": {
-        "jsx": true,
-      }
-    },
-    "plugins": [
-      "react",
-      "babel"
-    ],
-    "extends": ["eslint:recommended"],
-    "rules": {
-
-        // Lint Rules
-
-        "no-bitwise": 2,
-        "eqeqeq": 2,
-        "guard-for-in": 2,
-        "no-use-before-define": 2,
-        "no-caller": 2,
-        "no-new": 0,
-        "no-undef": 0, // Fix Me
-        "no-unused-vars": 0, // Fix Me
-        "strict": 0,
-        "max-params": [2, 5],
-        "semi": 0,
-        "no-cond-assign": 0,
-        "no-loop-func": 0,
-        "no-shadow": 0,
-        "no-underscore-dangle": 0,
-
-        // Style Rules
-
-        "no-with": 1,
-        "no-eval": 1,
-        "keyword-spacing": 1,
-
-        "curly": [1, "all"],
-        "space-before-blocks": 1,
-        "no-empty": 1,
-
-        "space-before-function-paren": [1, "never"],
-        "wrap-iife": 1,
-
-        "space-in-parens": [1, "never"],
-
-        "space-infix-ops": 1,
-        "dot-notation": 1,
-
-        "camelcase": 1,
-        "no-multi-str": 1,
-
-        "no-mixed-spaces-and-tabs": 1,
-        "no-trailing-spaces": 1,
-        "eol-last" : 1,
-
-        "comma-spacing": [1, {"before": false, "after": true}],
-        "indent": [1, 2],
-        "quotes": [1, "single"]
+  "extends": ["formidable/configurations/es6-react", "prettier"],
+  "env": {
+    "node": true,
+    "browser": true,
+    "mocha": true
```

Comment:
Maybe remove this and add a `test/.eslintrc` that extends this one plus adds it there?

Or, if bloating this PR, just separately ticket for later? (Not a huge priority)

Reply:
I am going to remove all test stuff and start fresh. But keep the spec files to ensure we're maintaining coverage when we move to jest.

Reply:
Sounds good!